ccpp/data/GFS_typedefs.meta

```diff
@@ -87,11 +87,11 @@
   standard_name = x_wind
   long_name = zonal wind
   units = m s-1
-  dimensions = (horizontal_loop_extent,vertical_dimension)
+  dimensions = (horizontal_loop_extent,vertical_layer_dimension)
   type = real
   kind = kind_phys
 [ugrs(:,1)]
-  standard_name = x_wind_at_lowest_model_layer
+  standard_name = x_wind_at_surface_adjacent_layer
```

Comment:
Why not "zonal" or "eastward"?

Reply:
Another good question. I think that the "x_wind*" names are a nod to different grids, and [x,y]_wind were deemed to be more general (work for a greater number of physics and hosts) than [eastward,northward]_wind. But, [x,y]_wind would also be deficient for, say, irregular or hexagonal grids too, so I'm not sure how valuable sticking with [x,y] really is.

CF standards do have [eastward,northward]_wind that we can use, though. If I remember correctly, FV3 converts winds on their C/D grid to true zonal/meridional winds on the A grid for the physics to use, correct? So, changing to [eastward,northward] should work in that case.

Reply:
See https://github.com/NCAR/ccpp-physics/discussions/692 for more discussion.